tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -359,21 +359,7 @@ def __init__(self, suite_name, index, test_queue,
         self.suite_name = suite_name
         self.manager_number = index
         self.test_implementation_key = None
-
-        self.test_implementations = {}
-        for key, test_implementation in test_implementations.items():
-            browser_kwargs = test_implementation.browser_kwargs
-            if browser_kwargs.get("device_serial"):
-                browser_kwargs = browser_kwargs.copy()
-                # Assign Android device to runner according to current manager index
-                browser_kwargs["device_serial"] = browser_kwargs["device_serial"][index]
-                self.test_implementations[key] = TestImplementation(
-                    test_implementation.executor_cls,
-                    test_implementation.executor_kwargs,
-                    test_implementation.browser_cls,
-                    browser_kwargs)
-            else:
-                self.test_implementations[key] = test_implementation
```

Comment:
Those lines are there so that Wptrunner can drive multiple emulators at the same time. Do we intentionally removed those lines?

Reply:
The indexing logic moved into `ChromeAndroidBrowserBase`.

Reply:
Ack

Reply:
(And `FirefoxAndroid`, etc. I didn't want to end up with more and more logic product-specific logic in the generic `testrunner`.)